README.md

```diff
@@ -1393,6 +1394,13 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Remote Desktop Tools
+
+  * [Zoho Assist](https://zoho.com/assist) — Free for 1 remote technician & 5 unattended computers
+  * [Getscreen.me](https://getscreen.me) —  Free for 2 devices, no limits on the number and duration of sessions
```

Comment:
Generally we avoid listing the same service in multiple categories, there is already a huge section dedicated to zoho, if anything assist should move in there rather than add a 3rd place zoho is listed.

That leaves this desktop category with just 1 entry and we dont accept 1 entry-only categories.  So maybe we can find a few more free screen sharing solutions?